code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -1,50 +1,46 @@
 /obj/item/gun/energy/laser
-	name = "laser gun"
+	name = "SL L-204 laser gun"
 	desc = "A basic energy-based laser gun that fires concentrated beams of light which pass through glass and thin metal."
 	icon_state = "laser"
 	item_state = "laser"
 	w_class = WEIGHT_CLASS_NORMAL
 	custom_materials = list(/datum/material/iron=2000)
 	ammo_type = list(/obj/item/ammo_casing/energy/lasergun)
 	ammo_x_offset = 1
-	shaded_charge = 1
+	shaded_charge = TRUE
 	supports_variations = VOX_VARIATION
+	manufacturer = MANUFACTURER_SHARPLITE_NEW
 
 /obj/item/gun/energy/laser/practice
 	name = "practice laser gun"
-	desc = "A modified version of the basic laser gun, this one fires less concentrated energy bolts designed for target practice."
+	desc = "A modified version of the L-204 laser gun, this one fires less concentrated energy bolts designed for target practice."
 	ammo_type = list(/obj/item/ammo_casing/energy/laser/practice)
-	clumsy_check = 0
+	clumsy_check = TRUE
 	item_flags = NONE
 
 /obj/item/gun/energy/laser/retro
-	name ="retro laser gun"
+	name ="SL L-104 retro laser gun"
 	icon_state = "retro"
-	desc = "An older model of the basic lasergun, no longer used by Nanotrasen's private security or military forces. Nevertheless, it is still quite deadly and easy to maintain, making it a favorite amongst pirates and other outlaws."
-	ammo_x_offset = 3
-
-/obj/item/gun/energy/laser/retro/old
-	name ="laser gun"
-	icon_state = "retro"
-	desc = "First generation lasergun, developed by Nanotrasen. Suffers from ammo issues but its unique ability to recharge its ammo without the need of a magazine helps compensate. You really hope someone has developed a better lasergun while you were in cryo."
-	ammo_type = list(/obj/item/ammo_casing/energy/lasergun/old)
+	desc = "An antiquated model of the basic lasergun, no longer used or sold by Sharplite. Nevertheless, the sheer popularity of this model makes it a somewhat common sight to this day."
 	ammo_x_offset = 3
+	manufacturer = MANUFACTURER_SHARPLITE
 
 /obj/item/gun/energy/laser/captain
 	name = "antique laser gun"
 	icon_state = "caplaser"
 	item_state = null
-	desc = "This is an antique laser gun. All craftsmanship is of the highest quality. It is decorated with assistant leather and chrome. The object menaces with spikes of energy. On the item is an image of Space Station 13. The station is exploding."
+	desc = "This is the SL X-00, an antique laser gun that none know how to recreate. All craftsmanship is of the highest quality. It is decorated with assistant leather and chrome. The gun menaces with spikes of energy."
```

Comment:
Slight wording update. Also readded the station image bit without actually naming SS13 for the sake of keeping the dwarf fortress gag intact. You can remove that if you want.

"This is the SL X-00, an antique laser gun, out of production for decades and well beyond anyone's capacity to recreate. All craftsmanship is of the highest quality. It is decorated with assistant leather and chrome. The gun menaces with spikes of energy. On the item is an image of a space station. The station is exploding."

Reply:
I would also like to argue that "assistant leather" be replaced with something less overtly bizarre, given Nanotrasen's marginally less deranged nature in our setting. Goliath leather, maybe.

Reply:
Would work, but with how common goliath leather is im not sure if that would be considered 'antique.' Maybe a more rare mob? cant think of one though

Reply:
Maybe a megafauna, like a drake?

Reply:
yeah, that would work